SU-HSDO/suhumsci
# STN-319: Sprint 14 QA Feedback

# READY FOR REVIEW

## Summary
- Created `hb-title` utility class
- Fixed button classes when used on a wrapper field
- Removed default external link icon, it can now be added with a class
- Fixed heading styles when alignment classes are added
- Updated `hs-font-lead` styles to be more distinctive

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Tests should be passing
2. Check that the issues listed in the [card](https://sparkbox.atlassian.net/browse/STN-319) have been fixed. _Note: the external link icon was going to be removed next sprint but we decided to remove it now since I was editing those styles._

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
